src/amo/components/RatingsByStar/styles.scss

```diff
@@ -32,6 +32,51 @@
 .RatingsByStar-barValue {
   background-color: rgba($yellow-50, 0.75);
 
+  // Pre-defined widths based on % of stars.
+  &.RatingsByStar-barValue--0pct {
```

Comment:
I can't think how right now, but it's a shame we can't do this with a sass mixin or something.

Reply:
something like:
```sass
@for $i from 0 through 10
    &.RatingsByStar-barValue--#{$i * 10}pct {
        width: 10% * $i;
    }
```

(completely untested!)

Reply:
Hmm, we might be able to. Let me try...

Reply:
Done!

Reply:
Actually, maybe I can improve on what I just pushed. One sec...